airbyte-connector-builder/openapi/generator_config.yaml

```diff
@@ -0,0 +1,4 @@
+files:
+  api_interfaces.mustache:
+    templateType: API
+    destinationFilename: _interface.py
```

Comment:
very small nit: missing newline at the end of the file

Reply:
fixed